tensorflow_lite_support/tools/Build_TensorFlow_Lite_Support_Libraries_with_Bazel_and_Colab.ipynb

```diff
@@ -0,0 +1,243 @@
+{
```

Comment:
<p>Can you please instructions for C++ targets as well?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tflite-support/pull/620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I did not catch this. Can you show me an example and the line of code?

Reply:
<p>The C++ targets are:</p><p> </p><p>bazel build -c opt \</p><p>&nbsp;&nbsp;&nbsp;//tensorflow_lite_support/cc/task/vision:image_classifier \ </p><p>   //tensorflow_lite_support/cc/task/vision:image_objector \ </p><p>   //tensorflow_lite_support/cc/task/vision:image_segmenter \ </p><p>   //tensorflow_lite_support/cc/task/vision:image_embedder \ </p><p>   //tensorflow_lite_support/cc/task/text/nlclassifier:nl_classifier \ </p><p>   //tensorflow_lite_support/cc/task/text/nlclassifier:bert_nl_classifier \ </p><p>   //tensorflow_lite_support/cc/task/text/qa:bert_question_answerer \ </p><p>&nbsp;&nbsp;//tensorflow_lite_support/metadata/cc:metadata_extractor \</p><p>&nbsp;</p>

Reply:
So the displayed name to the users will be "image_classifier" "image_objector"...just like "Support Library" "Task Vision" etc. ??

Reply:
What about "C++ image_classifier". "C++ object_detector", etc?

Reply:
OK